### ISSUE TITLE
Feature wish: Email notifaction in case of abnormal data

### ISSUE THREAD

**Issue by christophfriedrich** [sensebox/openSenseMap-API#53]
The registering user should be notified via email if their sensebox transmits unlikely data:
- very high or very low values
  - e.g. <-20 or >45°C temperature
  - possibly region depending
  - possibly season depending
- constant values for days
  - e.g. 100% humidity
  - good threshold IMO: three days?
- no transmission for days
  - i.e. when the status changes to inactive
  - or even earlier (again three days?)

That could help discover (and respond to) issues with ones sensebox earlier - when the two senseboxes I'm operating have issues I usually don't notice that for days or weeks as I don't manually check the OSeM very often.

**Comment by ubergesundheit** [comment]
Good idea. This and other notifications for users is definitely on our roadmap
